neuro-cli/src/neuro_cli/job.py

```diff
@@ -1467,9 +1479,11 @@ def _job_to_cli_args(job: JobDescription) -> List[str]:
         res += ["--privileged"]
     if job.priority != JobPriority.NORMAL:
         res += ["--priority", job.priority.name.lower()]
+    if job.energy_schedule_name:
+        res += ["--energy-schedule", job.energy_schedule_name]
     res += [str(job.container.image)]
     if job.container.command:
-        res += [job.container.command]
+        res += ["--", job.container.command]
```

Comment:
is this needed? maybe we should have a dedicated PR for this change?

Reply:
or this is super minor and won't possibly break anything?

Reply:
Haven't found that we use it elsewhere except our CLI (here only in generate_run_command).

This is mostly personal, I often use it to debug and the deprecation warning is annoying, decided to make this small fix in place :) Apologize, next time I'll open a dedicated PR for such things.